Pacifique78/MyDiary
# #169498323 fix delete entry endpoint errors

#### What does this PR do?
This fix delete-entries errors

#### Description of Task to be completed
>- Fix delete entry errors caused by finding the entry creator id
>- Change entry id to entry created by
>- Remove unused packages

#### How should this be manually tested?
>- clone the repo
>- run `npm install` and `npm run dev` to start the server
>- use either postman or any other testing tool to see if all endpoints are working 

#### What are the relevant pivotal tracker stories?
#169498323 

#### Screenshots (if appropriate)
